webextensions/api/tabs.json

```diff
@@ -3163,15 +3163,10 @@
                     "version_added": false
                   },
                   "edge": "mirror",
-                  "firefox": [
-                    {
-                      "notes": "Supports an array of strings to return tabs associated with any of the list of <code>cookieStoreId</code>s.",
-                      "version_added": "97"
-                    },
-                    {
-                      "version_added": "52"
-                    }
-                  ],
+                  "firefox": {
+                    "version_added": "52",
+                    "notes": "Since Firefox 97, this property supports an array of strings to return tabs associated with any of the list of <code>cookieStoreId</code>s."
```

Comment:
partial_implementation for the range before?

Reply:
It's hard to determine what qualifies as `partial_implementation` for web extensions, since there are no standards to refer to!  This was added in #13940.

Reply:
Maybe comparing to other browsers would help answer what's noteworthy and not? This note isn't too bad as it is (now with your edits) but one still wonders if this works in all browsers or not.

Reply:
No other browsers support this feature it looks like, so we have nothing to compare it to.  However, after reading the documentation on MDN, I've updated this with a partial implementation and rewrote the notes to indicate what it does and does not support.